docs/source/using/troubleshooting.adoc

```diff
@@ -194,3 +194,29 @@ flag in the `minishift start` command. Alternatively, the problem might be
 an outdated version of VirtualBox.
 
 To avoid this issue, it is recommended to use VirtualBox 5.1.12 or later.
+
+[[insufficient-privileges]]
+== Hyper-V: Administrator privileges insufficient to run Hyper-V
+
```

Comment:
Makes sense @hferentschik, I have got hold of a Windows machine for now, will verify this. I think @budhram is also trying it out.
I will add the missing pieces.